collections/_article/godot-4-0-sets-sail.md

```diff
@@ -0,0 +1,685 @@
+---
+title: "Godot 4.0 sets sail: All aboard for new horizons"
+excerpt: "Today marks the beginning of a new era for Godot. After 3+ years of breaking and rebuilding from the ground up, we're thrilled to say: Welcome to the start of Godot 4!" 
+categories: ["release"]
+author: "2000+ Godot contributors"
+image: /storage/blog/covers/godot-4-0-sets-sail.webp
+date: 2023-02-26 18:00:00
+---
+
+It's official. Today marks the beginning of a new era for Godot.
+
+After 3+ years of breaking and rebuilding from the ground up, a complete core overhaul and a full engine rewrite, through 17 alphas, 17 betas and 5 release candidates, we're thrilled to say:
```

Comment:
Just a reminder that the number of release candidates will need to change right before releasing